extensions/sql/policy/store/src/test/resources/schema.sql

```diff
@@ -0,0 +1,28 @@
+--
+--  Copyright (c) 2022 ZF Friedrichshafen AG - Initial API and Implementation
+--
+--  This program and the accompanying materials are made available under the
+--  terms of the Apache License, Version 2.0 which is available at
+--  https://www.apache.org/licenses/LICENSE-2.0
+--
+--  SPDX-License-Identifier: Apache-2.0
+--
+--  Contributors:
+--       ZF Friedrichshafen AG - Initial API and Implementation
+--
+
+-- table: edc_policies
+CREATE TABLE IF NOT EXISTS edc_policies
+(
+    policy_id VARCHAR(255) NOT NULL,
```

Comment:
The tests are meant to run with H2 DB, I think it is better to use VARCHAR(MAX) as it was defined it proposal. I just changed it to a certain int value to be able to run it with Postgres DB since Postgress doesn't have MAX and forget to change it back.

Reply:
why would we artificially constrain ourselves? H2 supports a `VARCHAR` datatype, same as Postgres. Seems a bit like having fixed-length `char[255]` in java code instead of `String`, no?
Now on some platforms (eg. MS SQL) `VARCHAR` has different semantics, so _there_ a length constraint makes sense, but as you said, we're targeting H2 and Postgres.